contrib/nydusify/pkg/checker/checker.go

```diff
@@ -108,6 +109,13 @@ func (checker *Checker) Check(ctx context.Context) error {
 	mode := "direct"
 	digestValidate := false
 
+	if sourceParsed.OCIImage.Manifest.Annotations[utils.LayerAnnotationNydusFsVersion] == "5" {
```

Comment:
Should be `sourceParsed.NydusImage` ?
And maybe the `--target` is not a nydus image, so better to check `sourceParsed.NydusImage != nil` here.

Reply:
Maybe we can implement a method named `NydusFsVersion()` for parser.Image to get this value, make it cleaner.

Reply:
Note: `Manifest.Annotations` could be `nil`.

Reply:
fixed